<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchExperimentWindow</class>
 <widget class="QDialog" name="BatchExperimentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch launch</string>
  </property>
  <widget class="QComboBox" name="par_log">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="index_log">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>510</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="time_log">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>69</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>-1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="run_but">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>510</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run All</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>561</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>510</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="import_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_but">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
